index.html

```diff
@@ -33,6 +33,105 @@ <h1 class="header2">I’m Iptisam
+
+            <div class="container3">
+                <div class="rect3"></div>
+                <h4 class="Proj3">Tonic</h4>
+                <ul class="list3">
+                    <li class="frame2"><img src="Images/Frame2.png" alt="frame"></li>
+                </ul>
+                <p class="text3">A daily selection of privately personalized reads; no accounts or sign-ups required.</p>
+                <ul class="Lang2">
+                    <li class="HTML2"><a href="#">html</a></li>
+                    <li class="css2"><a href="#">CSS</a></li>
+                    <li class="Java2"><a href="#">Javascript</a></li>
+                </ul>
+                <ul class="box2">
+                <li class="button2"><a href="#">See Project</a></li>
+                </ul>
+            </div>
+
+            <div class="container4">
+                <div class="rect4"></div>
+                <h5 class="Proj4">Multi-Post Stories</h5>
+                <ul class="list4">
+                    <li class="frame3"><img src="Images/Frame2.png" alt="frame"></li>
+                </ul>
+                <p class="text4">A daily selection of privately personalized reads; no accounts or sign-ups required.</p>
+                <ul class="Lang3">
+                    <li class="HTML3"><a href="#">html</a></li>
+                    <li class="css3"><a href="#">CSS</a></li>
+                    <li class="Java3"><a href="#">Javascript</a></li>
+                </ul>
+                <ul class="box3">
+                <li class="button3"><a href="#">See Project</a></li>
+                </ul>
+            </div>
+            <div class="container5">
+                <h6 class="header3">About Myself</h6>
+                <p class="descrip">I’m a software developer! I can help you build a product , feature or website Look through some of my work and experience! If you like what you see and have a project you need coded, don’t hestiate to contact me.</p>
+                <p class="links1">LET’S CONNECT</p>
+                <ul class="images1">
+                    <li class="shape12"><a href="#"><img src="Images/Frame1.png" alt="shape"></a></li>    
+                </ul>
+                <ul class="box4">
+                    <li class="button4"><a href="#">Get my resume</a></li>
+                </ul>
+                <nav class="About">
+                    <ul>
+                        <li class="LANG"><a href="#">Languages</a></li>
+                        <li class="arr1"><a href="#"><img src="Images/Icon.png" alt="arrow"></a></li>
+                        <li class="FRAME"><a href="#">Frameworks</a></li>
+                        <li class="arr2"><a href="#"><img src="Images/Icon1.png" alt="arrow"></a></li>
+                        <li class="SKILL"><a href="#">Skills</a></li>
+                        <li class="arr3"><a href="#"><img src="Images/Icon1.png" alt="arrow"></a></li>
+                    </ul>
+                    <ul class="Lang4">
```

Comment:
- [ ] Kindly check the styling of this section the icons are misplaced:
![image](https://user-images.githubusercontent.com/61408860/185751025-1b8d4983-30d9-4247-94c9-23706f15f386.png)